### ISSUE TITLE
CowGame: change scoring and 'Game Over' message color in a dark mode

### ISSUE THREAD

**Issue by elena-zh** [gnosis/cowswap#914]
Scoring and 'Game Over' message is in black and is hardly visible when switch to the dark mode. 
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/5666b240-dd90-4462-b73a-f26480cd70b4)

It would be nice to change scoring and 'Game Over' message color to rgb(197, 218, 239)

**Comment by alfetopito** [comment]
cc @biocom although I agree not high priority

**Comment by biocom** [comment]
I agree too. Requires more effort given these characters are all sprite (pixel) based. I think I need to choose a non-black color which fits both (e.g. orange or something else...). Will address.
